### PR TITLE
Update wpf-xaml-browser-applications-overview.md

### PR DESCRIPTION
**Pull request by Trevorthe37th**
Corrected ambiguity of supported browsers and to align with the Internet Explorer supportability documentation as "iemode" uses iexplore.exe.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Review by TrevorFellman-MSFT — APPROVED** [review]
Looks good

**Comment by adegeo** [comment]
Can you provide more context about this change? I've never heard anyone refer to Firefox or IE as "Desktop Application"

**Comment by TrevorFellman-MSFT** [comment]
The request came from CSS - Browsers support.
This is to disambiguate from the new options.  I am open to changing the wording.

IE Desktop App = yes
IE Mobile = no
IE Mode (Chrome, Edge) = no
WebBrowser control (WPF/Winforms) or mshtml = no
Old Firefox Desktop app = yes 
New Firefox Desktop app = no
Firefox for Android = no
Firefox for IOS = no
Firefox for Mac = no
All other browsers = no

image

File: dotnet-desktop-guide/framework/wpf/app-development/wpf-xaml-browser-applications-overview.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ ms.assetid: 3a7a86a8-75d5-4898-96b9-73da151e5e16
 # WPF XAML Browser Applications Overview
 
 <a name="introduction"></a>
-XAML browser applications (XBAPs) combines features of both Web applications and rich-client applications. Like Web applications, XBAPs can be deployed to a Web server and started from Internet Explorer or Firefox. Like rich-client applications, XBAPs can take advantage of the capabilities of WPF. Developing XBAPs is also similar to rich-client development. This topic provides a simple, high-level introduction to XBAP development and describes where XBAP development differs from standard rich-client development.
+XAML browser applications (XBAPs) combines features of both Web applications and rich-client applications. Like Web applications, XBAPs can be deployed to a Web server and started from Internet Explorer Desktop Application or Firefox Desktop Application. Like rich-client applications, XBAPs can take advantage of the capabilities of WPF. Developing XBAPs is also similar to rich-client development. This topic provides a simple, high-level introduction to XBAP development and describes where XBAP development differs from standard rich-client development.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/app-development/wpf-xaml-browser-applications-overview.md:19]:
```suggestion
XAML browser applications (XBAPs) combines features of both Web applications and rich-client applications. Like Web applications, XBAPs can be deployed to a Web server and started from Internet Explorer or Firefox on Windows. Like rich-client applications, XBAPs can take advantage of the capabilities of WPF. Developing XBAPs is also similar to rich-client development. This topic provides a simple, high-level introduction to XBAP development and describes where XBAP development differs from standard rich-client development.

>[!IMPORTANT]
> XBAPs require legacy browsers to operate, such as Internet Explorer and Firefox. These older browsers are usually unsupported on Windows 10 and Windows 11.
```

**Review by adegeo — CHANGES_REQUESTED** [review]
What do you think of this?

I just stated "on Windows" instead of specifying "Desktop Application" which is more to the point, Windows only. I also added a note below that talks about how XBAPs aren't really supported anymore because the browser and OS technology has moved on.

File: dotnet-desktop-guide/framework/wpf/app-development/wpf-xaml-browser-applications-overview.md (hunk 1 above)
Comment (TrevorFellman-MSFT) [reply dotnet-desktop-guide/framework/wpf/app-development/wpf-xaml-browser-applications-overview.md:19]: That is even better!  I really like the "important" note, it spells out the reality rather concisely.

File: dotnet-desktop-guide/framework/wpf/app-development/wpf-xaml-browser-applications-overview.md [hunk 2]
```diff
@@ -16,7 +16,10 @@ ms.assetid: 3a7a86a8-75d5-4898-96b9-73da151e5e16
...
+XAML browser applications (XBAPs) combines features of both Web applications and rich-client applications. Like Web applications, XBAPs can be deployed to a Web server and started from Internet Explorer or Firefox on Windows. Like rich-client applications, XBAPs can take advantage of the capabilities of WPF. Developing XBAPs is also similar to rich-client development. This topic provides a simple, high-level introduction to XBAP development and describes where XBAP development differs from standard rich-client development.
+
+>[!IMPORTANT]
+> XBAPs require legacy browsers to operate, such as Internet Explorer and Firefox. These older browsers are usually unsupported on Windows 10 and Windows 11.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/app-development/wpf-xaml-browser-applications-overview.md:22]:
```suggestion
> XBAPs require legacy browsers to operate, such as Internet Explorer and Firefox. These older browser versions are usually unsupported on Windows 10 and Windows 11.
```

**Review by adegeo — APPROVED** [review]

**Pull request merged**